docs-chef-io/content/server/config_rb_server_optional_settings.md

```diff
@@ -1461,6 +1461,12 @@ This configuration file has the following settings for `oc-id`:
 
     **New in Chef Infra Server 15.9.20**
 
+`oc_id['onetrust_id']`
+
+: The Onetrust ID for Chef Infra. You must also set `oc_id['enable_onetrust']` to `true`.
+
+  Default value: `nil`.
+
```

Comment:
I'm not sure which version this is 👇 
```suggestion

  **New in Chef Infra Server X.Y.Z**

```

Reply:
@sreepuramsudheer @kalroy Which release version would this change go in?
I can add the version accordingly.